stdlib/stdlib.mli

```diff
@@ -1065,6 +1065,30 @@ module LargeFile :
   regular integers (type [int]), these alternate functions allow
   operating on files whose sizes are greater than [max_int]. *)
 
```

Comment:
Wouldn't it make more sense to put all these aliases much earlier in the signature, in particular before they are used?